module_utils/ansible_nailgun_cement.py

```diff
@@ -187,6 +186,14 @@ class ComputeProfile(ComputeProfile, entity_mixins.EntitySearchMixin):
     pass
 
 
+class ConnectionCheck(Entity, entity_mixins.EntitySearchMixin):
```

Comment:
I don't know why `EntitySearchMixin` would be required here, this API is not searchable. I think that an error in nailgun as well https://github.com/SatelliteQE/nailgun/blob/master/nailgun/entities.py#L4615

Reply:
Turns out EntitySearchMixin is required so we can do a `GET` to it.